build.gradle

```diff
@@ -75,7 +75,7 @@ allprojects {
     }
 
     // Version has to be set directly here, using a variable does not set Java compatability version
-    sourceCompatibility = 1.8
+    sourceCompatibility = 1.9
```

Comment:
:question: Open Question :question: 
How do we want to handle Java versions with [Semantic Versioning](http://semver.org/spec/v2.0.0.html)?
Upgrading from Java 8 to Java 9 could be considered breaking change (major), however Java 9 is something that would be valuable to get into the 5.x release line.

Reply:
My thought is would be to set a lower target compatibility allowing this to be added in a feature (minor) release.

```gradle
sourceCompatibility = 1.9
targetCompatibility = 1.8
```

Then removing the backwards target in uPortal 6.